compiler/crates/relay-transforms/tests/relay_test_operation/fixtures/test_client_fields_query.expected

```diff
@@ -0,0 +1,37 @@
+==================================== INPUT ====================================
+query ClientFieldsQuery @relay_test_operation {
+  node(id: "test-id") {
+    id
+    ... on User {
+      name
+      client_info {
+        name
+        description
+      }
+    }
+  }
+}
+
+# %extensions%
+
+extend type User {
+  client_info: ClientInfo
+}
+
+type ClientInfo {
+  name: String
+  description: String
+}
+==================================== OUTPUT ===================================
+query ClientFieldsQuery @__metadata(relayTestingSelectionTypeInfo: {node: {enumValues: null, nullable: true, plural: false, type: "Node"}, node.id: {enumValues: null, nullable: false, plural: false, type: "ID"}, node.name: {enumValues: null, nullable: true, plural: false, type: "String"}, node.client_info: {enumValues: null, nullable: true, plural: false, type: "ClientInfo"}, node.client_info.name: {enumValues: null, nullable: true, plural: false, type: "String"}, node.client_info.description: {enumValues: null, nullable: true, plural: false, type: "String"}}) {
```

Comment:
this is the new behavior with the feature flag enabled. This extra metadata will allow `MockPayloadGenerator` to auto-mock client fields when they're not provided in the associated resolver